tsconfig.json

```
@@ -21,9 +21,7 @@
     "jsx": "preserve",
     "baseUrl": ".",
     "paths": {
-      "@/components/*": [
-        "components/*"
-      ]
+      "@/*": ["./*"],
```

Trailing commas aren't proper JSON, even if the interpreter tolerates them
```suggestion
      "@/*": ["./*"]
```